scripts/spawn-workstream-windows.sh

```
@@ -0,0 +1,364 @@
+# `run_dry` is a thin wrapper for dry-run echoing ONLY. Callers pass a
+# descriptive string; real execution is inlined at the call site using
+# an argv array (never `eval`) to avoid shell-injection via variable
+# interpolation of externally-derived values (branch names, paths, etc.).
+dry_log() { echo "DRY: $*"; }
+
+[[ -f "$WORKSTREAMS_DOC" ]] || fail "workstreams doc not found at $WORKSTREAMS_DOC"
+[[ -f "$SETTINGS_LOCAL" ]] || fail "settings.local.json not found at $SETTINGS_LOCAL — create it before spawning"
+
+CODE_BIN=""
+for c in /opt/homebrew/bin/code /usr/local/bin/code; do
+  [[ -x "$c" ]] && CODE_BIN="$c" && break
```

Shellcheck (0.11.0)

[warning] CODE_BIN appears unused. Verify use (or export if used externally).

(SC2034)